src/components/HistoryCard/ViHistoryCard.css

```diff
@@ -0,0 +1,83 @@
+
+:root {
+    --hover-color: #f3f1f1f3;
+    --shadow-color: black;
+    --card-subheadcolor: gray;
+    --icon-hover:#f5dadaf3;
+
+}
+
+.card {
+    width: 20rem;
+    display: flex;
+    align-items: center;
+    flex-direction: column;
+}
+
+.card:hover {
+    background-color: var(--hover-color);
+}
+
+.image img {
+    width: 100%;
+    border-top-left-radius: 0.5rem;
+    border-top-right-radius: 0.5rem;
+}
+
+.title {
+    margin-left: 0.5rem;
+    /* margin-top: 0.5rem; */
+
+}
+
+.title__header {
+    padding-bottom: 4px;
+}
+
+.title__subheading,
+.content__color {
+   color: var(--card-subheadcolor);
+}
+
+.content {
+    margin-left: 1rem;
+    margin-top: 1rem;
+    margin-bottom: 1rem;
+    flex-wrap: wrap;
+}
+.card_footer {
+    display: flex;
+    justify-content: space-around;
+    border: none;
+    align-items: center;
+}
+
+.footer_info {
+    display: flex;
+    justify-content: space-around;
+    margin-left: auto;
+}
+
+.btn-footer {
+    font-size: large;
+    font-weight: 100;
+    margin-right: 1rem;
+}
+
+.btn-footer:hover {
+    background-color: var(--hover-color);
+
+}
+
+.footer_icons {
+    font-size: 2rem;
+}
+
+.vicons {
```

Comment:
Use better classname, not sure what this reffers to.

Reply:
okay